layers/+distributions/spacemacs-base/funcs.el

```diff
@@ -588,12 +588,14 @@ dotspacemacs-persistent-server to be t"
 (defun spacemacs/kill-emacs ()
   "Lose all changes and exit Spacemacs"
   (interactive)
+  (run-hook-with-args-until-failure 'kill-emacs-query-functions)
   (setq spacemacs-really-kill-emacs t)
   (kill-emacs))
 
 (defun spacemacs/prompt-kill-emacs ()
   "Prompt to save changed buffers and exit Spacemacs"
   (interactive)
+  (run-hook-with-args-until-failure 'kill-emacs-query-functions)
   (setq spacemacs-really-kill-emacs t)
   (save-some-buffers)
   (kill-emacs))
```

Comment:
I don't know, I figured I'd fix this and solve that problem later. There is presumably a reason because elsewhere we *do* call `save-buffers-kill-emacs`.